src/test_l1_dsAudio.c

```diff
@@ -5495,16 +5495,23 @@ void test_l1_dsAudio_positive_dsSetAudioDelay(void)
         result = dsGetAudioPort(gDSAudioPortConfiguration[i].typeid, gDSAudioPortConfiguration[i].index, &handle);
         UT_ASSERT_EQUAL(result, dsERR_NONE);
         UT_ASSERT_NOT_EQUAL(handle, null_handle);
+        if(gDSAudioPortConfiguration[i].typeid != dsAUDIOPORT_TYPE_HEADPHONE)
```

Comment:
If there is another audio port which doesn't support delay will fail again. So, instead of adding check for headphone, add check for delay supported ports mentioned in header file
if(gDSAudioPortConfiguration[i].typeid == HDMI || gDSAudioPortConfiguration[i].typeid == ARC || gDSAudioPortConfiguration[i].typeid == SPDIF)